main.py

```diff
@@ -1,49 +1,33 @@
-# Load std packages
-import os
-import zipfile
-import pandas as pd
-import numpy as np
-#import PIL.Image
-import time
-import seaborn as sns
-import matplotlib.pyplot as plt
-#from tqdm import tqdm_notebook
-#%matplotlib inline
-import cv2
-from tqdm.auto import tqdm
-
 # custom ones
-from utils import load, plot_result #@todo : add other paquet once the training is over
-from preprocessing.preprocessing import greyscale_to_rgb, one_hot_y, split_train_test, resize, normalize_img
-from models.model import model_toto, model_toto2, model_toto3, model_v4, history_save
+from utils import load, preprocess_img, plot_result  # @todo : add other paquet once the training is over
+from preprocessing.preprocessing import one_hot_y, split_train_test
+from models.model import history_save, model_v4
+from include import RUN_LOCAL
```

Comment:
import inutile

Reply:
why? I use almost all of the functions above.